cantherm/chemistry/molecule.py

```diff
@@ -73,9 +73,17 @@ def __init__(self, file_path: str):
 
         # Optional properties
         try:
```

Comment:
I think we should ditch this `try` block and switch to an `if` block, which would be safer. It's my fault for using an unsafe `try`/`except` statement in the first place. I think the new block should look something like this:

```python
            
            self.vibfreqs = getattr(self.data, "vibfreqs", np.array([]))
            if self.vibfreqs.size > 0:
                self.vibfreqs = self.vibfreqs[self.vibfreqs > 0]
                self.imag_vibfreqs = self.vibfreqs[self.vibfreqs > 0]
            else:
                self.imag_vibfreqs = np.array([])
                print(f"No vib data found for {file_path}")
                print(f"{file_path} will not be used for vibrational contributions")

```

Reply:
I'm not used to conditionals phrased like this but I've tentatively used what you had except with `self.imag_vibfreqs = self.vibfreqs[self.vibfreqs < 0]` Hopefully that's correct

Reply:
:point_up: that's totally right I messed up and used greater than